src/main/app/src/pages/HomePage/useFilterState.ts

```diff
@@ -44,122 +17,146 @@ export const useFilterState = (name: ENTITY) => {
       previousOrganisaatioOid != null &&
       selectedOrganisaatioOid !== previousOrganisaatioOid
     ) {
-      setPagination({ page: 0 });
+      send({ type: 'RESET_PAGINATION' });
```

Comment:
Jos oikein ymmärsin, niin tämä resetoi nyt sivutuksen kaikille entiteeteille kyseisessä filterMachine:ssa. Ilmeisesti tämä ei ole näin aiemmin toiminut? On kyllä hieman hämmentävää, vaikka ei tässä ilmeisesti väärin toimikaan.